test/integration/kueuectl/list_test.go

```diff
@@ -72,14 +74,17 @@ var _ = ginkgo.Describe("Kueuectl List", ginkgo.Ordered, ginkgo.ContinueOnFailur
 
 			kueuectl.SetArgs([]string{"list", "localqueue", "--field-selector",
 				fmt.Sprintf("metadata.name=%s", lq1.Name), "--namespace", ns.Name})
+			executeTime := time.Now()
 			err := kueuectl.Execute()
 
 			gomega.Expect(err).NotTo(gomega.HaveOccurred(), "%s: %s", err, output)
 			gomega.Expect(errOutput.String()).Should(gomega.BeEmpty())
 
-			gomega.Expect(output.String()).Should(gomega.Equal(`NAME   CLUSTERQUEUE   PENDING WORKLOADS   ADMITTED WORKLOADS   AGE
-lq1    cq1            0                   0                    0s
-`))
+			gomega.Expect(output.String()).Should(gomega.Equal(fmt.Sprintf(`NAME   CLUSTERQUEUE   PENDING WORKLOADS   ADMITTED WORKLOADS   AGE
```

Comment:
IIUC the issue is that `output.String()` may return 1s rather than 0s as typically. However, I think it could still flake if `time.Now()` uses a different time than `output.String()` . I guess they would need to return different second, which is unlikely, but possible. 

I think it is better to use `gomega.Eventually` to re-try if the times differ. And call `time.Now` and `output.String()` within the lambda.

Alternatively, we could maybe use a fake clock, but I'm not sure how much refactoring this would be.

Reply:
Yes, you right, now this issue in progress #2287. So in follow-up I think we can make it.

Reply:
Ok, so please don't just close the issue https://github.com/kubernetes-sigs/kueue/issues/2353. We will close it when the follow up is ready. In the PR description say "Part of #2353"

Reply:
And maybe let's say in title "Make the "should print workloads list with paging" test less likely to fail"